error.controllers.js

```diff
@@ -12,6 +12,11 @@ exports.handleEmptyEntry = (err, req, res, next) => {
         res.status(400).send({ msg: "No input detected"})}
     else next(err);
 }
+exports.handleInvalidArticleIdComment = (err, req, res, next) => {
```

Comment:
There are LOADS of psql errors

you don't _need_ a seperate handler for each `err.code` - you could handle them all in the same one and have if/else/switch statements to determine which status/message to send